<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin CP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Transactions</string>
     </attribute>
     <widget class="QPushButton" name="loadAllTransactionsButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load All Transactions</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="transactionsTable">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>461</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="editTransactionRowButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Selected Row</string>
      </property>
     </widget>
     <widget class="QLabel" name="invalidRowSelectedMsg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color:red
};</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>215</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="salesDateLabel">
               <property name="text">
                <string>Sales Day:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="salesDayInput"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="showSalesButton">
             <property name="text">
              <string>Show Sales</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Member ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="memberIDField"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="showSalesByMemberIDButton">
           <property name="text">
            <string>Show Sales</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Items</string>
     </attribute>
     <widget class="QTableView" name="itemsView">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>11</y>
        <width>461</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>251</width>
        <height>146</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="itemNameLabel">
           <property name="text">
            <string>Item Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="itemNameField"/>
         </item>
         <item>
          <widget class="QLabel" name="itemPriceLabel">
           <property name="text">
            <string>Item Price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="itemPriceField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="deleteItemButton">
           <property name="text">
            <string>Delete Item</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addItemButton">
           <property name="text">
            <string>Add Item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="customersPage">
     <attribute name="title">
      <string>Customers</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>371</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>12</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Expiring Memberships</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>340</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
